extensions/fairifier-rdf-extension/dtls/fairifier/DetectFileFormatCommand.java

```diff
@@ -83,8 +89,7 @@ public void doPost(HttpServletRequest req, HttpServletResponse res)
         // Create a new file upload handler
         final ServletFileUpload upload = new ServletFileUpload(factory);
         String baseuri = null;
-        String filename = null;
-        InputStream[] in = new InputStream[parsers.length];
+        InputStream[] in = new InputStream[parsers.size()];
```

Comment:
Not sure if we still need this array approach, but we can leave it for now.

Reply:
We need it because if we parse full files the streams might be closed on an exception